Header header
int32 totalImages
int32 missedImages
int32 avg_ftime
int32 avg_fps